content/docs/implementation-notes.md

```diff
@@ -394,36 +394,36 @@ As a result, each internal instance, composite or host, now points to its child
 }
 ```
 
-In the DOM you would only see the `<div>`. However the internal instance tree contains both composite and host internal instances.
+Dans le DOM, vous voyez uniquement le `<div>`. Cependant l’arbre d’instance interne contient des instances internes de composite et d'hôte.
 
-The composite internal instances need to store:
+Les instances internes de composite doivent stocker :
```

Comment:
```suggestion
Les instances internes composites ont besoin de stocker :
```

- Adjectif…
- J'aime pas quand on traduit “need to“ par « doit ». Ce n'est pas la même perspective (orientée besoin vs. orientée contrainte). Je vois ça fréquemment dans les trads proposées, je sais pas d'où ça vient.